manifests/init.pp

```diff
@@ -99,19 +102,23 @@
   Variant[Boolean[false], Pattern[/icmp(v6|x)? type .+|tcp reset/]] $reject_with = 'icmpx type port-unreachable',
   Variant[Boolean[false], Enum['mask']] $firewalld_enable = 'mask',
   Optional[Array[Pattern[/^(ip|ip6|inet)-[-a-zA-Z0-9_]+$/],1]] $noflush_tables = undef,
+  Stdlib::Unixpath $configuration_path = '/etc/sysconfig/nftables.conf',
```

Comment:
can you move this to hiera as well? If we generate the docs with puppet-strings, the data from the init.pp will land in the REFERENCE.md. People will then assume that `/etc/sysconfig/nftables.conf` will be used on all systems.

Reply:
For sure